### DISCUSSION TITLE
multiple workers?

### DISCUSSION THREAD

**Discussion by mustafa0x** [RealOrangeOne/django-tasks#51] [Q&A] [answered]
The readme mentions "multiple worker processes", but I couldn't find any other indication in the docs and code indicating the library does indeed support multiple workers.

**Comment by RealOrangeOne** [comment, marked as answer]
Multiple workers is supported by running the `db_worker` management command multiple times. Any process management etc should be managed externally. Each process will only run 1 task at a time, so for increased throughput you'll need to run multiple worker processes.
